src/functionalTest/java/uk/gov/hmcts/probate/functional/caseprogress/SolCcdCaseProgressAdmonWillTests.java

```diff
@@ -1435,6 +1436,41 @@ public void shouldTransformCaseCorrectlyWhenMarkingReadyToIssue() {
         assertEquals(expected, taskList);
     }
 
+    @Test
+    public void shouldTransformCaseCorrectlyWhenFailQa() {
+        final String response = postCwJson("caseprogressadmonwill/06-caseStopped.json", CASE_FAIL_QA_URL);
+        final JsonPath jsonPath = JsonPath.from(response);
+        final String taskList = jsonPath.get("data.taskList");
+
+        String expectedHtml = "<div class=\"width-50\">\n\n<h2 class=\"govuk-heading-l\">Case progress</h2>\n\n"
```

Comment:
you should put this expected response into some resource file + read it as in other tests